andela/AndelaEatsUI
# #160479776 Implement edit vendor functionality

**What does the PR do?**
- Implements `Update Vendor` functionality

**Description of Task to be completed?**
- Ensures an admin can update a vendor details

**How should this be manually tested?**
- Clone the repository [here](https://github.com/andela/AndelaEatsUI.git)
-  cd into AndelaEatsUI and checkout to this branch `ft-impement-edit-vendor-functionality-160479776`
- run `npm run bundle` and `npm run dev`
- Log in as an `Admin`, navigate to `admin dashboard`, select `Vendors`, and click on `edit`
- run `npm test` to ensure all tests are passing

**What are the relevant pivotal tracker stories?**
- [#160479776](https://www.pivotaltracker.com/story/show/160479776)

**Screenshots**
<img width="1680" alt="screen shot 2018-09-18 at 2 47 28 am" src="https://user-images.githubusercontent.com/26305570/45660755-56728d80-baf2-11e8-90dc-1aa899ac3948.png">
<img width="1676" alt="screen shot 2018-09-18 at 2 47 42 am" src="https://user-images.githubusercontent.com/26305570/45660761-5d010500-baf2-11e8-9e0d-dc83ac1ba8af.png">
